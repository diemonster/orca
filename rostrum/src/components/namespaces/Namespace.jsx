import React from 'react';

import NamespaceList from './NamespaceList';
import NamespaceCreate from './NamespaceCreate';
import NamespaceDelete from './NamespaceDelete';

function Namespace() {
  return (
    <div className='kube-display'>
      <NamespaceList />
      <NamespaceCreate />
      <NamespaceDelete />
    </div>
  );
}

export default Namespace;
