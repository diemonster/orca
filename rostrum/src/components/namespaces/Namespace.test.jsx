import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Namespace from './Namespace';

Enzyme.configure({ adapter: new Adapter() });

describe('components', () => {
  describe('Namespace', () => {
    it('should render itself and its subcomponents', () => {
      const namespace = shallow(<Namespace />);
      expect(namespace).toMatchSnapshot();
    });
  });
});
