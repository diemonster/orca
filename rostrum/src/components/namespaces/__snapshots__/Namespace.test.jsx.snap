// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components Namespace should render itself and its subcomponents 1`] = `
<div
  className="kube-display"
>
  <Connect(NamespaceList) />
  <Connect(NamespaceCreate) />
  <Connect(NamespaceDelete) />
</div>
`;
